libs/src/services/creatorNode/index.js

```diff
@@ -836,6 +847,15 @@ class CreatorNode {
         console.warn(`Network Error in request ${requestId} with ${retries} retries... retrying`)
         console.warn(e)
         return this._uploadFile(file, route, onProgress, extraFormDataOptions, retries - 1)
+      } else if (e.response && e.response.data && e.response.data.error && e.response.data.error.includes('Invalid authentication token')) {
```

Comment:
we can prob factor this out into a `reconnect` function and call it - or move into `ensureConnected()`, since it's the same in both places.

Reply:
+1